src/main/thrift/octopus.thrift

```diff
@@ -0,0 +1,112 @@
+namespace scala com.gu.flexibleoctopus.model.thrift
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+/** 
+* A single story bundle that maps directly to a piece of content in Composer 
+*/
+struct StoryBundle {
+
+    1: required i64 id;
+
+    2: required string composerId;
+
+    3: required string pubCode;
+
+    4: required string sectionCode;
+
+    5: required Article article;
+
+}
+
+/** The legal statues of an article */
+enum Lawyered {
+    NOTAPPLICABLE = 0,
+    PENDING = 1,
+    CLEARED = 2,
+    PRIORITY = 3 
+}
+
+/** Where an article is going to be published */
+enum ForPublication {
+    WEB = 0,
+    PRINT = 1,
+    BOTH = 2
+}
+
+/** The workflow statuses of the article */ 
+enum ArticleStatus {
+    CHIEFSUB = 0,
+    DESK = 1,
+    FINALLED = 2,
+    HOLD = 3,
+    KILLED = 4,
+    REVISESUB = 5,
+    SUBS = 6,
+    WRITERS = 7
+}
+
+/** The object types that can be present in an article */
+enum ArticleObjectType {
```

Comment:
I'm not entirely convinced on this one. Could we omit object type entirely, if we are only ever looking for body copy? Or would we want to include the other types, on the basis that we might use them in the future?

Reply:
I could imagine us wanting to show/use (Daily?) eg print headline one day.

Reply:
Yeah, and (sorry to be morbid but) there's the "8 feared dead ... 10 feared dead ... 15 feared dead" type of story where the Composer sub may want to know that a print headline that was finalled is now out of date.

Reply:
@hoyla are those story bundle statuses?

Reply:
@jennygrahamjones I'm more thinking that one day we _might_ want to show the **content** of a print headline in workflow so a sub could know that the content of that headline is no longer appropriate. If you excuse the ramble, I think this might be a good example for illustrating a potential workflow... So, bearing in mind that a big part of what we're trying to accomplish here is to minimise the time spent doing substantive work in the print production system for digital first content, imagine that in some desks (not all) we have nearly all content work done in Composer by one person, and then in print someone **else** is effectively just managing a page layout and cutting articles to an exact fit. In this workflow the person on the print side will not be expected to know a great deal about the actual content of the articles. So if the print headline is now wrong, the Composer sub is more likely to know that than the page layout person. Therefore showing the print headline in Workflow allows the subject matter expert in Composer to see that it's wrong, and consequently warn the layout person that they should change it. This isn't a feature request for now, I'm just imagining things that might be useful in future and therefore echoing @paperboyo's suggestion that we may eventually not want to discard object types that aren't body text.

Reply:
The current datamodel is such that rather than an array of articles we flatten these into the main struct as options named after the object type (I guess on the basis that a bundle never has two of the same thing)?

As such this enum isn't needed as it is pre-processed during conversion from JSON.